### PR TITLE
feat: disable community settings while in Testnet mode

### LINKED ISSUE #11468: Disabling Community token related functionality while in Testnet mode
While in Testnet mode, the user should not be able to access the Community admin area. This helps prevent the user from deploying and interacting with Community smart contracts while in this mode.

Designs have been created showing what the Community founder, Owner (if Owner token minted), TokenMaster (if TokenMaster token minted and airdropped) and admin should see when they launch the Community admin area.

- The entire community admin nav is disabled
- The body area is taken over with a panel explaining that the admin area is disabled with a CTA to turn off Testnet mode
- Testnet mode can be turned off either via the above CTA or the omnipresent orange Testnet banner
- If Testnet is turned off while in this view, the UI will refresh and show the user their last visited admin area location with the orange Testnet banner removed

Figma designs:
https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=34976-596521&mode=design&t=7HSjl1pkHU0oNfhu-4

Testnet1

Testnet2

Testnet3

### PR DESCRIPTION
**Pull request by caybro**
### What does the PR do

- display an info box when in wallet testnet mode with a CTA to disable
it (functionality enabled for production builds only)
- the CTA can be tested/seen via OverviewSettingsPanelPage storybook page
- plus separate commits to add support for icon in `StatusInfoBox` and fix consistent usage of its `warning` color

Closes #11468

### Affected areas

CommunitySettingsView, OverviewSettingsPanel, StatusInfoBox

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

[Záznam obrazovky z 2023-07-24 13-27-35.webm](https://github.com/status-im/status-desktop/assets/5377645/b06e708e-bdba-438b-a809-a4a0efeade73)

Testnet info banner (with Community editable off):
image

### CODE REVIEW

**Review by micieslak — APPROVED** [review]
Looks nice in general, I understand the motivation for the used approach. I would probably do it differently to avoid altering `OverviewSettingsPanel` and to avoid loading the stack layout and it's content (settings sections). It would require to keep separate model for left panel (I know - I merged it for convenience with stack layout's children but it doesn't fit that case completely). Then centerPanel could be loader loading stacklayout or placeholder for testnet mode.

**Review by alexandraB99 — APPROVED** [review]

File: ui/app/AppLayouts/Communities/panels/OverviewSettingsPanel.qml [hunk 1]
```diff
@@ -142,6 +142,35 @@ StackLayout {
         }
     }
 
+    Component {
+        id: disabledSettingsBannerComp
+        StatusInfoBoxPanel {
```
Comment (alexjba) [inline ui/app/AppLayouts/Communities/panels/OverviewSettingsPanel.qml:147]: The layout is a little bit different in Figma. The width is static, to 560 and height dynamic based on content.

App:
Screenshot 2023-07-24 at 18 07 27

Design:
Tokens

**Review by alexjba — APPROVED** [review]
Minor comment on panel size. Other than that LGTM! 👍

File: ui/app/AppLayouts/Communities/panels/OverviewSettingsPanel.qml (hunk 1 above)
Comment (caybro) [reply ui/app/AppLayouts/Communities/panels/OverviewSettingsPanel.qml:147]: Fixed, nice spot! (also fixed the padding/radius)

**Pull request merged**